test/adapters/kagome/cmake/HunterConfig.cmake

```diff
@@ -17,6 +17,11 @@
 
 # TODO: Move away from custom kagome branch for testing
 hunter_config(kagome
-    URL https://github.com/soramitsu/kagome/archive/af5c718a05a3b734291dd2c4f574c8dffb76ad46.tar.gz
-    SHA1 81f456bb92d06e7b5861dacb66d4ff77515dadf6
+    URL https://github.com/soramitsu/kagome/archive/219d46ff77ae8b77974373653ce25e720ef0645f.tar.gz
+    SHA1 1a0827b729b4dc1e40e37887df53f05bbcac3607
 )
+hunter_config(libsecp256k1
+    URL https://github.com/soramitsu/soramitsu-libsecp256k1/archive/c7630e1bac638c0f16ee66d4dce7b5c49eecbaa5.zip
+    SHA1 179e316b0fe5150f1b05ca70ec2ac1490fe2cb3b
+    CMAKE_ARGS SECP256K1_BUILD_TEST=OFF
+    )
```

Comment:
This should not be required as it is not a direct dependency.

Reply:
Nevermind, my bad. I had to find out the hard way that _libsecp256k1_ otherwise misses a `VERSION` field.